scripts/e2e.sh

```diff
@@ -20,14 +22,14 @@ echo "e2e_tower_license: '"$TOWER_LICENSE"'" >> e2e_test_config.yml
 echo "e2e_tower_password: '"$TOWER_PASSWORD"'" >> e2e_test_config.yml
 echo "e2e_tower_username: '"$TOWER_USERNAME"'" >> e2e_test_config.yml
 
-echo "Bootstrapping tower in openshift project: $tower_openshift_project"
+echo "Bootstrapping tower in openshift project: ${OPENSHIFT_MASTER}"
 ansible-playbook -i ./inventories/hosts \
 playbooks/bootstrap.yml \
 --extra-vars "@e2e_test_config.yml"
 bootstrap_exit_code=$?
 echo "Bootstrap playbook ansible exit code: $bootstrap_exit_code"
 
 
-oc delete project $tower_openshift_project
+oc delete project ${OPENSHIFT_MASTER}
```

Comment:
Is this correct? Should it not be TOWER_OPENSHIFT_PROJECT?

Reply:
🤦‍♀